apps/backend/package.json

```diff
@@ -15,7 +15,7 @@
     "build": "nest build",
     "lint": "eslint \"{src,migrations,seeders,test}/**/*.ts\" --fix",
     "lint:ci": "eslint \"{src,migrations,seeders,test}/**/*.ts\" --max-warnings 0",
-    "sequelize-cli": "ts-node --esm node_modules/.bin/sequelize",
+    "sequelize-cli": "node --loader ts-node/esm node_modules/.bin/sequelize",
```

Comment:
what does this do?

Reply:
instead of calling ts-node directly, use the loader flag to still use node as the runner but with ts-node as the loader

also changing the flag name from --loader to --experimental-loader since it was apparently changed to that